src/to-be-empty.js

```diff
@@ -1,6 +1,10 @@
 import { matcherHint } from 'jest-matcher-utils';
 import { checkReactElement, isEmpty, printElement } from './utils';
 
+/**
+ * @deprecated This function is deprecated. You should use `toBeEmptyElement`
+ *
+ * */
 export function toBeEmpty(element) {
   checkReactElement(element, toBeEmpty, this);
 
```

Comment:
Pls make following changes:
1. Move this function to `to-be-empty-element.js` file
2. Make `toBeEmpty` simply call `toBeEmptyElement`
3. Log deprecation message redirecting users to the new matcher to the console when deprecated matcher is called for the first time.

Reply:
For deprecation message you can use following code

```ts
const warned: { [functionName: string]: boolean } = {};

export function printDeprecationWarning(message: string) {
  if (warned[functionName]) {
    return;
  }

  // eslint-disable-next-line no-console
  console.warn(`
  Deprecation Warning:
  ${message}:
  `);

  warned[functionName] = true;
}
```

then call it
```ts
printDeprecationWarning(`"toBeEmpty()" matcher has been renamed to "toBeEmptyElement()". Old name will be deleted in future versions of @testing-library/jest-native.`)
```